### PR TITLE
Add displaying games of selected genre

### PR DESCRIPTION
**Pull request by AleksandroBoko**
When genre page is loaded by the first time the list of the genre's games will be shown for user.

### CODE REVIEW

File: GameStore/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -97,10 +97,17 @@
         })
         strResult += "</select><div id=ganreGamesTable></div></div>";
         $("#tableBlock").html(strResult);
+        
+        if (genres != null && genres.length > 0) {
```
Comment (SafeNuts) [inline GameStore/Views/Home/Index.cshtml]: If I am not making a mistake, you can perform such check just as boolean in C#.
`if(genres && genres.length > 0) {`

File: GameStore/Views/Home/Index.cshtml [hunk 2]
```diff
@@ -97,10 +97,17 @@
...
+            let genre = document.getElementById("genre");
+            if (genre !== null) {
```
Comment (SafeNuts) [inline GameStore/Views/Home/Index.cshtml]: Same as above

File: GameStore/Views/Home/Index.cshtml (hunk 1 above)
Comment (AleksandroBoko) [reply GameStore/Views/Home/Index.cshtml]: Optimized

File: GameStore/Views/Home/Index.cshtml (hunk 2 above)
Comment (AleksandroBoko) [reply GameStore/Views/Home/Index.cshtml]: Optimized

**Pull request merged**